domain/src/main/java/org/oppia/domain/profile/ProfileManagementController.kt

```diff
@@ -144,25 +149,24 @@ class ProfileManagementController @Inject constructor(
       val nextProfileId = it.nextProfileId
       val profileDir = directoryManagementUtil.getOrCreateDir(nextProfileId.toString())
 
-      val imageUri: String?
+      val newProfileBuilder = Profile.newBuilder()
+        .setName(name).setPin(pin).setAllowDownloadAccess(allowDownloadAccess)
```

Comment:
Small nit: but it's probably cleaner to keep each '.setX()' on its own line.

Reply:
Updated.